src/bridge/secrets/xqwatcher/secrets.mitxonline.production.yaml

```
@@ -1,12 +1,39 @@
 ---
 graders_yaml:
   graders:
+  - name: ENC[AES256_GCM,data:Saeklgo3kKMLB3BhMwp+YW4UhSzKgEm41wm2,iv:xzG67ZjgSkMshCqdY+QmO2/lY0ptt7fPHXomOEPtFEo=,tag:ObZqTiG11lQ9TKC/EOyQ3Q==,type:str]
+    address: ENC[AES256_GCM,data:oGfjw+e3JlUIJWtk5bhynWoFRMr1X8ZrPPgcCIZT51cvb86IcTr5+g==,iv:BHtztNt6TsNggmMuz4J6uXLWZvNc80gmiitwg7AkPeI=,tag:yiDViNSjKX3xiRfLRGg7LQ==,type:str]
+    git_ref: ENC[AES256_GCM,data:RNxgY5Rv,iv:hILWee8Dj5/C5YBWQh8v1icKpIgZcZSS3ZeqEuFeZns=,tag:IWM2eydyNjS0GV5f1MEg/Q==,type:str]
+    env:
+      GIT_SSH_COMMAND: ENC[AES256_GCM,data:VXlhmOm/sq6i2Po2lbqOY0NIiie40yLuMqe8VrfIlUgVUk5zlFOY1pa0BkMZGzVDP8nT2BwaM8zyeqSnOtdkX4moi5lQYP86FuD1wkZiPx4APgIY0pgiJMuvSparZKw/D/uIu8kPxXeUsCAdde/7gN9Nxe99FOz+mOQEYIpG66Zjyj7l0XpqrAcza3fBwMDn,iv:Li4i2qmdGqBeCsTd2zsoccS4wZ4yFize4zm4TfRHr+Y=,tag:fqt6NvzloFvLuEZ8AdFo8g==,type:str]
   - name: ENC[AES256_GCM,data:CJEFoJq2E2Q=,iv:/svfF2BhTRufD4HUY4/e3U4C1uoZg93i59mdkd0NTqg=,tag:gpeMihlWF3szCMlFnhtk+w==,type:str]
     address: ENC[AES256_GCM,data:GnVPPRFinmGb/CdTHNitpfqXShU3mgghcFmDoY2QqZGTaI8Tm75z6A==,iv:U2MmVWQjrflNpLKMGb5+DgnL2bk99M0DpSu/7APpjsU=,tag:jCZs5mMBrYqOs9nyU7wxcg==,type:str]
     git_ref: ENC[AES256_GCM,data:RNxgY5Rv,iv:hILWee8Dj5/C5YBWQh8v1icKpIgZcZSS3ZeqEuFeZns=,tag:IWM2eydyNjS0GV5f1MEg/Q==,type:str]
     env:
       GIT_SSH_COMMAND: ENC[AES256_GCM,data:VXlhmOm/sq6i2Po2lbqOY0NIiie40yLuMqe8VrfIlUgVUk5zlFOY1pa0BkMZGzVDP8nT2BwaM8zyeqSnOtdkX4moi5lQYP86FuD1wkZiPx4APgIY0pgiJMuvSparZKw/D/uIu8kPxXeUsCAdde/7gN9Nxe99FOz+mOQEYIpG66Zjyj7l0XpqrAcza3fBwMDn,iv:Li4i2qmdGqBeCsTd2zsoccS4wZ4yFize4zm4TfRHr+Y=,tag:fqt6NvzloFvLuEZ8AdFo8g==,type:str]
 confd_json:
+  residential-6.00x:
```

```suggestion
  Watcher-MITx-6.00x:
```
For consistency with QA